### DISCUSSION TITLE
Received this error when trying to clone a repository to my windows 11 laptop.

### DISCUSSION THREAD

**Discussion by MrKwamiD-86** [community/community#75575] [New to GitHub] [answered]
Cloning into 'C:\Users\adank\Documents\TWR2019-2023'...
remote: Enumerating objects: 132, done.        
remote: Counting objects: 100% (72/72), done.        
remote: Compressing objects: 100% (63/63), done.        
remote: Total 132 (delta 23), reused 40 (delta 1), pack-reused 60        
Receiving objects: 100% (132/132), 22.46 KiB | 4.49 MiB/s, done.
Resolving deltas: 100% (45/45), done.
      0 [main] sh (4320) C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x1371408/0x1321408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
      0 [main] sh 677 dofork: child -1 - forked process 4320 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (1856) C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x1371408/0x1171408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
1167436 [main] sh 677 dofork: child -1 - forked process 1856 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (15172) C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x1371408/0x12E1408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
3317780 [main] sh 677 dofork: child -1 - forked process 15172 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (14808) C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x1371408/0x1121408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
7487862 [main] sh 677 dofork: child -1 - forked process 14808 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      1 [main] sh (17196) C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x1371408/0x1391408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
15654326 [main] sh 677 dofork: child -1 - forked process 17196 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\adank\AppData\Local\GitHubDesktop\app-3.3.5\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: Resource temporarily unavailable

This error keeps me from accessing the repository via the github desktop app, and in turn, prevents me from working locally. Thanks for any help/insight. 
P.s. I am able to run github desktop fine on my windows 11 desktop. Just not so with my laptop. ☹️

**Comment by MrKwamiD-86** [comment, marked as answer]
Solved the problem! 
Had to disable Mandatory ASLR.

To do this:
Go to 'Windows Security' > Select 'App & Browser Control' > Select 'Exploit Protection Settings' > Find 'Force Randomization For Images (Mandatory ASLR)' > Select 'Off' > Restart computer.
:)

**Reply by drkdiniz** [reply]
Hi, I had the same problem as you and I managed to solve it by following this tutorial. Thank you very much.

**Reply by Exstreme-Noob** [reply]
Had the same problem and it worked. Thank you very much.
